main.cpp

```diff
@@ -278,12 +278,13 @@ int main (int argc, char** argv)
 				std::cerr << "Error dispatching: " << retval << std::endl;
 			else if (retval != 0)
 				packets_read = true;
+			else
```

Comment:
There is a small problem with that.
When packets are received, `curtime` is updated inside the callbakc called by `pcap_dispath`
The bandwith calculation is done inside `do_refresh()` but it uses